### DISCUSSION TITLE
Can we have release tag for this tool

### DISCUSSION THREAD

**Discussion by niladripapu** [microsoft/Windows-Driver-Developer-Supplemental-Tools#54] [Ideas]
Can youplease create release tag for this repo which is compatible of  CodeQL CLI 2.4.6 ?
So that CodeQL-Queries  will be backward compatible .
In the CI/CD evironment users are not getting time to update but getting broken main line very frequently .

**Comment by NateD-MSFT** [comment]
Hi there,

The WHCP_21H2 branch continues to be compatible with CodeQL 2.4.6.  Is this sufficient for your needs?

**Reply by niladripapu** [reply]
Thanks for the reply , I will test my job and update you ,
regards
